### ISSUE TITLE
No GitHub repositories found

### ISSUE THREAD

**Issue by SimonBiggs** [ReviewNB/jupyterlab-gitplus#4]
I'm full of these :). New issue, on both server and local machine I get the following error:

![image](https://user-images.githubusercontent.com/6559099/79283531-84e96200-7efb-11ea-8008-6432728f5274.png)

Might this be because I am using an organisation on GitHub (pymedphys) instead of my own personal org?

**Comment by amit1rrr** [comment]
@SimonBiggs Are you opening the files you want to commit in JupyterLab? 

We look at opened files in JupyterLab & decide which of those are under a GitHub repository. 

> Might this be because I am using an organisation on GitHub (pymedphys) instead of my own personal org?

No, this shouldn't matter.

**Comment by SimonBiggs** [comment]
Yup, the changed files were open :)

**Comment by amit1rrr** [comment]
Just asking a couple of questions here in order to debug,

1. Can you drop to command line, `cd` into where the file is, and run `git remote -v`? What's the output?

2. Can you open the inspect tab in the chrome browser where JupyterLab is running (right click -> inspect). Head over to network tab in the inspect window and see if the request is sending correct file paths for all open files? I have attached screenshots below to help see the steps.

![Screenshot 2020-04-15 at 9 51 19 AM](https://user-images.githubusercontent.com/1350490/79298633-4eade180-7eff-11ea-8edc-252eb70aacf5.png)

![Screenshot 2020-04-15 at 9 52 05 AM](https://user-images.githubusercontent.com/1350490/79298641-53729580-7eff-11ea-9266-a76814ed5077.png)

**Comment by SimonBiggs** [comment]
Yup, I can do that. I'll have to do it tomorrow as I'm heading home now.

**Comment by tekumara** [comment]
I have the same problem. 

Jupyter lab logs show:
```
File is not a part of GitHub repository: /Users/tekumara/code/torch-threads/torch-threads.ipynb
```

```
$ cd ~/code/torch-threads
$ git remote -v
origin	git@github.com:tekumara/torch-threads.git (fetch)
origin	git@github.com:tekumara/torch-threads.git (push)
```

Request payload in developer tools to http://localhost:8888/gitplus/modified_repo shows 200 OK and:
```
{"files":[{"path":"/Users/oliver.mannion/code/torch-threads/torch-threads.ipynb"}]}
```

**Comment by amit1rrr** [comment]
@tekumara Thank you for providing the details. I see [this line](https://github.com/ReviewNB/jupyterlab-gitplus/blob/master/jupyterlab_gitplus/handlers.py#L18) is causing the error now.

I will release a fix in the next few hours. Sorry for the trouble.

**Comment by amit1rrr** [comment]
@tekumara @SimonBiggs 

I have release a fix in `0.1.15` of server side package. Can you please update with,

`pip install --upgrade jupyterlab_gitplus`

Feel free to reopen the issue if you see any problem.

**Comment by SimonBiggs** [comment]
No need to be sorry :). The pleasure is ours :). This is a new release, we
are the first few using it apart from yourself, rough edges to be expected.
